### PR TITLE
pkgctl: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
Sorry. Didn't know deleting a repo deletes merge requests as well. I pushed to main in my own fork and thought to reset by just deleting the fork. Original here https://github.com/tldr-pages/tldr/pull/10523

File: pages/linux/pkgctl.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# pkgctl
+
+> Unified command-line devtools frontend for Arch Linux.
+> More information: <https://man.archlinux.org/man/pkgctl.1>.
+
+- Download PKGBUILD of a package:
```
Comment (mfrw) [inline pages/linux/pkgctl.md:6]: Does it create a new directory or places `PKGBUILD` etc in the directory in which it was invoked ?

Comment (mfrw) [reply pages/linux/pkgctl.md:6]: I think we might as well add that, what do you think ?

Comment (Managor) [reply pages/linux/pkgctl.md]: There. That should be good.

Comment (mfrw) [reply pages/linux/pkgctl.md]: Thank you :)

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @Managor for your contribution.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> I will merge this PR for now and work on other subcommand pages later.

**Pull request merged**